hack/ci/arm

```diff
@@ -4,7 +4,7 @@ set -eu -o pipefail
 
 hack/test/unit
 
-hack/make.sh \
+TEST_SKIP_INTEGRATION_CLI=1 hack/make.sh \
```

Comment:
`TEST_SKIP_INTEGRATION_CLI = '1'` is added in `environment` section of Jenkinsfile pipeline, so this modification is not needed.
In fact, the instruction here is moved to `"Integration tests"` stage of Jenkinsfile, the whole file `hack/ci/arm` is no longer needed.